src/common/hooks/useScrollHandler.tsx

```diff
@@ -0,0 +1,16 @@
+import { useEffect } from 'react';
```

Comment:
created this hook since you were watching for scroll events in two places, I ended up pulling that into one place and left the hook.